### PR TITLE
chore: erc20 contract call tests

### PR DESCRIPTION
**Pull request by nmlinaric**
## Description
unit tests for new erc20 and native token handles optional contract call

### CODE REVIEW

File: test/handlers/erc20/optionalContracCall/executeProposal.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+      DefaultMessageReceiverInstance.address
+    );
+
+    await ERC20MintableInstance.approve(
+      ERC20HandlerInstance.address,
+      depositAmount,
+      {from: depositorAddress}
+    );
+
+    // eslint-disable-next-line max-len
```
Comment (mpetrun5) [inline test/handlers/erc20/optionalContracCall/executeProposal.js:111]: I'm doubtfull this is needed.

Comment (mpetrun5) [reply test/handlers/erc20/optionalContracCall/executeProposal.js:111]: There is a couple of them which are unnecessary.

**Review by mpetrun5 — APPROVED** [review]
I guess it looks good. 
Just a small nitpick.

**Review by lastperson — CHANGES_REQUESTED** [review]
This is good.
We also need tests for the default message receiver.

**Review by lastperson — APPROVED** [review]

**Pull request merged**